### DISCUSSION TITLE
Doe PostGrest Support 32 bit machine?

### DISCUSSION THREAD

**Discussion by paramThakkar1505** [PostgREST/postgrest#3626] [Q&A] [answered]
There are some of our clients having 32 bit machines, so will it work with 32 bit machine?

**Comment by wolfgangwalther** [comment, marked as answer]
You will have to build postgrest yourself - we don't provide any 32bit binaries.

Once you did and tested it - please let us know whether it worked or not.

**Reply by paramThakkar1505** [reply]
How can I build it?

**Reply by wolfgangwalther** [reply]
Depending on which platform you are building on you can try via:
- nix: https://github.com/PostgREST/postgrest/blob/main/nix/README.md
- stack: https://postgrest.org/en/v12/explanations/install.html#building-from-source

**Reply by paramThakkar1505** [reply]
Did your older versions support 32 bit Windows platform?

**Reply by wolfgangwalther** [reply]
The first release for Windows that I can find is https://github.com/PostgREST/postgrest/releases/tag/v0.3.0.1. This only has a 64-bit binary attached.

I don't think anyone ever tried.
